docs/guide/sorting-data.md

```diff
@@ -1,17 +1,202 @@
 # Sorting data
 
+In HyperFormula, you can sort data by reordering rows and columns.
+
+## Sorting data in HyperFormula
+
+Sorting data in HyperFormula works similarly to Google Sheets and Microsoft Excel:
+
+1. First, HyperFormula sorts formulas by their results (calculated values).
+2. When the sorting is done, each formula is recalculated with regard to its new position.
```

Comment:
Nope, that's description for Handsontable sorting. HF does not compare data, you just provide new order. How it was calculated is up to the user

Reply:
@wojciechczerniak - ok, makese sense - so the previous intro was just a placeholder then. How about this: 9df4703

Reply:
Perfect 👍